### PR TITLE
Delete mapStruckt 

### PR DESCRIPTION
**Pull request by M437A**

### CODE REVIEW

File: src/main/java/com/zufar/icedlatte/payment/converter/PaymentConverter.java [hunk 1]
```diff
@@ -2,19 +2,69 @@
...
+
+        return processedPaymentDetailsDto;
+    }
+
+    protected Set<ShoppingCartItemDto> shoppingCartItemSetToShoppingCartItemDtoSet(Set<ShoppingCartItem> set) {
+        if ( set == null ) {
+            return null;
+        }
+
+        Set<ShoppingCartItemDto> set1 = new LinkedHashSet<ShoppingCartItemDto>( Math.max( (int) ( set.size() / .75f ) + 1, 16 ) );
```
Comment (annstriganova) [inline src/main/java/com/zufar/icedlatte/payment/converter/PaymentConverter.java:46]: nit: what are these magic numbers? Could you extract them to constants?

**Review by annstriganova — APPROVED** [review]
Approved.

**Comment by annstriganova** [comment]
I believe `org.mapstruct` dependency should be removed from `pom.xml` as well.

File: src/main/java/com/zufar/icedlatte/payment/converter/PaymentConverter.java (hunk 1 above)
Comment (M437A) [reply src/main/java/com/zufar/icedlatte/payment/converter/PaymentConverter.java:46]: if be honest, i dnk, i just made a copy of mapStruct's class)

**Pull request closed without merging**